test/new-e2e/pkg/utils/clients/ssh.go

```diff
@@ -78,3 +80,80 @@ func ExecuteCommand(client *ssh.Client, command string) (string, error) {
 
 	return string(stdout), err
 }
+
+// CopyFile create a sftp session and copy a single file to the remote host through SSH
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Did you consider using `scp` instead ? https://github.com/bramvdbogaerde/go-scp

Reply:
I briefly had a look to both library, I could not notice a big difference between both. `scp` has some higher level function but still does not support recursively copying files OOTB. So I decided to use `sftp` which is the most used package and is maintained by Go.